src/algokit/cli/project/bootstrap.py

```diff
@@ -102,5 +102,11 @@ def poetry() -> None:
 @bootstrap_group.command(
     "npm", short_help="Runs `npm install` in the current working directory to install Node.js dependencies."
 )
-def npm() -> None:
-    bootstrap_npm(Path.cwd())
+@click.option(
+    "--ci",
+    is_flag=True,
+    default=lambda: "CI" in os.environ,
```

Comment:
Confirming this is automatically set on a GitHub actions environment?

Reply:
Yes it is, see https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#default-environment-variables